ember-client/src/main/scala/org/http4s/ember/client/internal/ClientHelpers.scala

```diff
@@ -105,7 +113,7 @@ private[client] object ClientHelpers {
           Parser.Response
             .parser(maxResponseHeaderSize, durationToFinite(timeout))(
               head,
-              connection.keySocket.socket.read(chunkSize, durationToFinite(idleTimeout))
+              connection.keySocket.socket.read(chunkSize)
```

Comment:
Was deleting this timeout what we want?

Reply:
We could just add the timeout on the effect now

Reply:
Yep.  It's a nuisance because that takes a `FiniteDuration` instead of a `Duration`.  And the `socket.reads` and `socket.writes` in ember-server aren't as easy to string that through.